install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -55,35 +55,35 @@ spec:
+              value: registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.4.0
             - name: STRIMZI_DEFAULT_KAFKA_EXPORTER_IMAGE
-              value: registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.3.0
+              value: registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.4.0
             - name: STRIMZI_DEFAULT_CRUISE_CONTROL_IMAGE
-              value: registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.3.0
+              value: registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.4.0
             - name: STRIMZI_KAFKA_IMAGES
               value: |
-                3.2.3=registry.redhat.io/amq7/amq-streams-kafka-32-rhel8:2.3.0
-                3.3.1=registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.3.0
+                3.2.3=registry.redhat.io/amq7/amq-streams-kafka-32-rhel8:2.4.0
+                3.3.1=registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.4.0
             - name: STRIMZI_KAFKA_CONNECT_IMAGES
               value: |
-                3.2.3=registry.redhat.io/amq7/amq-streams-kafka-32-rhel8:2.3.0
-                3.3.1=registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.3.0
+                3.2.3=registry.redhat.io/amq7/amq-streams-kafka-32-rhel8:2.4.0
+                3.3.1=registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.4.0
             - name: STRIMZI_KAFKA_MIRROR_MAKER_IMAGES
               value: |
-                3.2.3=registry.redhat.io/amq7/amq-streams-kafka-32-rhel8:2.3.0
-                3.3.1=registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.3.0
+                3.2.3=registry.redhat.io/amq7/amq-streams-kafka-32-rhel8:2.4.0
+                3.3.1=registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.4.0
             - name: STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES
               value: |
-                3.2.3=registry.redhat.io/amq7/amq-streams-kafka-32-rhel8:2.3.0
-                3.3.1=registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.3.0
+                3.2.3=registry.redhat.io/amq7/amq-streams-kafka-32-rhel8:2.4.0
+                3.3.1=registry.redhat.io/amq7/amq-streams-kafka-33-rhel8:2.4.0
             - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
-              value: registry.redhat.io/amq7/amq-streams-rhel8-operator:2.3.0
+              value: registry.redhat.io/amq7/amq-streams-rhel8-operator:2.4.0
             - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
-              value: registry.redhat.io/amq7/amq-streams-rhel8-operator:2.3.0
+              value: registry.redhat.io/amq7/amq-streams-rhel8-operator:2.4.0
             - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
-              value: registry.redhat.io/amq7/amq-streams-rhel8-operator:2.3.0
+              value: registry.redhat.io/amq7/amq-streams-rhel8-operator:2.4.0
             - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
-              value: registry.redhat.io/amq7/amq-streams-bridge-rhel8:2.3.0
+              value: registry.redhat.io/amq7/amq-streams-bridge-rhel8:2.4.0
```

Comment:
I see you are also moving/renaming folders from 3.2.3 to 3.4.0 so I guess we should make changes here as well. 
Using the 3.4.0 as default one with images like amq-streams-kafka-34, replacing the 3.2.3 and images like amq-streams-kafka-32.